docs/reference/cli.md

```diff
@@ -115,18 +94,17 @@ quickwit index create
 
 *Examples*
 
-*Create a new index with metastore in the local `metastore` folder.*
+*Create a new index with the default local metastore.*
 ```bash
-quickwit index create --index-config-uri ./wikipedia_index_config.yaml  --config=quickwit.yaml
+quickwit index create --index wikipedia --index-config wikipedia_index_config.yaml  --config=./config/quickwit.yaml
```

Comment:
I prefer it without subfolder since it's simpler for the user
`--config=quickwit.yaml`

Reply:
I assume the user have a fresh install of quickwit and thus has the YAML file in the config dir. I think both are ok.

Reply:
I don't understand how a new install is related to having the config file in the subfolder